static/src/js/front/form.js

```diff
@@ -1,15 +1,23 @@
 (function (window, document, undefined) {
 
+    var uri_base = '';
+    if (typeof noviusos_form_params !== 'undefined') {
```

Comment:
Would have set `noviusos_form_params === 'object'` to be sure that it's an object.